test/ssl-tests/protocol_version.pm

```diff
@@ -257,8 +257,7 @@ sub generate_resumption_tests {
                 $resumption_expected = "No";
             }
 
-            for (my $sctp = 0; $sctp < ($dtls && !disabled("sctp") ? 2 : 1);
-                 $sctp++) {
+            for (my $sctp = 0; $sctp < 1; $sctp++) {
```

Comment:
```suggestion
```
Delete this line and the matching closing parenthesis and unindent the enclosed body.

Reply:
I was going to do this next after someone else double-checked the loop I changed :)